apps/frontend/index.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="UTF-8" />
         <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+        <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
```

Comment:
@afolasope It looks like this is from a personal website and not a CDN, so it reliability is not guaranteed. Can you use https://fonts.googleapis.com/css?family=Inter instead?

Reply:
> @afolasope It looks like this is from a personal website and not a CDN, so it reliability is not guaranteed. Can you use https://fonts.googleapis.com/css?family=Inter instead?

I found it in the tailwind documentation but i have changed it now